src/sentry/sentry_metrics/consumers/indexer/parallel.py

```diff
@@ -1,111 +1,180 @@
 from __future__ import annotations
 
-import functools
+import logging
 from functools import partial
-from typing import Callable, Mapping, Union
+from typing import Callable, Mapping, Optional, Union
 
 from arroyo.backends.kafka import KafkaConsumer, KafkaPayload
 from arroyo.processing import StreamProcessor
-from arroyo.processing.strategies import ProcessingStrategy, ProcessingStrategyFactory
+from arroyo.processing.strategies import ProcessingStrategy
+from arroyo.processing.strategies import ProcessingStrategy as ProcessingStep
+from arroyo.processing.strategies import ProcessingStrategyFactory
 from arroyo.processing.strategies.streaming.transform import ParallelTransformStep
-from arroyo.types import Partition, Position, Topic
+from arroyo.types import Message, Partition, Position, Topic
 from django.conf import settings
 
+from sentry.runner import configure
 from sentry.sentry_metrics.configuration import MetricsIngestConfiguration
-from sentry.sentry_metrics.consumers.indexer.common import BatchMessages, get_config
+from sentry.sentry_metrics.consumers.indexer.common import BatchMessages, MessageBatch, get_config
 from sentry.sentry_metrics.consumers.indexer.multiprocess import SimpleProduceStep
 from sentry.sentry_metrics.consumers.indexer.processing import process_messages
 from sentry.utils.batching_kafka_consumer import create_topics
 
+logger = logging.getLogger(__name__)
 
-@functools.lru_cache(maxsize=10)
-def get_metrics():  # type: ignore
-    from sentry.utils import metrics
 
-    return metrics
+class Unbatcher(ProcessingStep[MessageBatch]):  # type: ignore
+    """
+    Temporary Transform Step
+    """
 
```

Comment:
This is not temporary anymore. Sorry, this was the code I was using for the load test, I did not have the time to clean it up.